src/main/connector/database/MySQL/MySQL.yaml

```diff
@@ -0,0 +1,297 @@
+extends:
+- ../Database/Database
+connector:
+  displayName: MySQL
+  platforms: Any platform running MySQL
+  reliesOn : MySQL Database version 8.0 or higher
+  information: Monitors performance and operational metrics for a MySQL database.
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - windows
+    - linux
+    criteria:
+    - type: sql
+      query: SELECT @@version_comment REGEXP 'mysql' AS is_mysql;
+      expectedResult: 1
+      errorMessage: Not a MySQL Server
+    tags: [ linux, windows, database ]
+metrics:
```

Comment:
Move all the metric definitions to Database.yaml and keep only mysql specific metrics here.